web_src/css/actions.css

```diff
@@ -44,7 +44,7 @@
 }
 
 .run-list-item-right {
-  flex: 0 0 min(20%, 130px);
+  width: 130px;
```

Comment:
Maybe it could be `max(20%, 130px)`?

Reply:
I found static width looked better there.

Reply:
Maybe the only width which needs to be limited is for the "branch name". 

Other parts like "run-list-item-right" could remove the hard-coded width and let the UI layout automatically?

Reply:
As was requested, I added a static width on it for 300px on desktop size now. Much better than any percentage-based width because that will cause unwantend content shifting.